### PR TITLE
Use foundry for integration tests

### PR DESCRIPTION
**Pull request by Schaeff**
- Remove solc dependency
- Generate a single foundry project for all integration tests
- Run `forge test` at the end

To be squashed before merging

### CODE REVIEW

File: zokrates_cli/tests/integration.rs [hunk 1]
```diff
@@ -59,6 +60,38 @@ mod integration {
...
+            .unwrap();
+
+        std::io::stdout().write_all(&output.stdout).unwrap();
+        std::io::stderr().write_all(&output.stderr).unwrap();
+
+        assert!(output.status.success());
+
+        Command::new("rm")
+            .current_dir(&solidity_test_path)
+            .arg("./src/Counter.sol")
```
Comment (dark64) [inline zokrates_cli/tests/integration.rs]: can probably use a wildcard here `.arg("./src/*.sol")` and remove the other call below

**Review by dark64 — COMMENTED** [review]
small suggestion

File: zokrates_cli/tests/integration.rs (hunk 1 above)
Comment (Schaeff) [reply zokrates_cli/tests/integration.rs]:
```suggestion
            .arg("./src/*.sol")
```

File: zokrates_cli/tests/integration.rs [hunk 2]
```diff
@@ -59,6 +60,38 @@ mod integration {
...
+            .output()
+            .unwrap();
+        Command::new("rm")
+            .current_dir(&solidity_test_path)
+            .arg("./test/Counter.t.sol")
```
Comment (Schaeff) [inline zokrates_cli/tests/integration.rs]:
```suggestion
            .arg("./test/*.t.sol")
```

**Pull request merged**